packaging/build_wheels.bash

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+set -xe
+# A script to loop over the available pythons installed
+# on Linux/OSX and build wheels
+#
+# Note: It should be invoked from nmodl directory
+#
+# PREREQUESITES:
+#  - cmake (>=3.3)
+#  - flex
+#  - bison
+#  - python >= 3.6
+#  - C/C++ compiler
+
+if [ ! -f setup.py ]; then
+    echo "Error: setup.py not found. Please launch $0 from the nmodl root dir"
+    exit 1
+fi
+
+setup_venv() {
+    local py_bin="$1"
+    local py_ver=$("$py_bin" -c "import sys; print('%d%d' % tuple(sys.version_info)[:2])")
+    local venv_dir="nmodl_build_venv$py_ver"
+
+    if [ "$py_ver" -lt 36 ]; then
+        echo "[SKIP] Python $py_ver not supported"
+        skip=1
+        return 0
+    fi
+
+    echo " - Creating $venv_dir: $py_bin -m venv $venv_dir"
+    "$py_bin" -m venv "$venv_dir"
+    . "$venv_dir/bin/activate"
+
+    # pep425tags are not available anymore from 0.35
+    # temporary workaround until we get smtg stable
+    if ! pip install -U pip setuptools "wheel<0.35"; then
+        curl https://bootstrap.pypa.io/get-pip.py | python
+        pip install -U setuptools "wheel<0.35"
+    fi
+}
+
+
+build_wheel_linux() {
+    echo "[BUILD WHEEL] Building with interpreter $1"
+    local skip=
+    setup_venv "$1"
+    (( $skip )) && return 0
+
+    echo " - Installing build requirements"
+    pip install pip auditwheel setuptools scikit-build
+    pip install -r packaging/build_requirements.txt
+
+    echo " - Building..."
+    rm -rf dist _skbuild
+    python setup.py bdist_wheel
+
+    if [ "$TRAVIS" = true ] ; then
+        echo " - Skipping repair on Travis..."
+        mkdir wheelhouse && cp dist/*.whl wheelhouse/
+    else
+        echo " - Repairing..."
+        auditwheel repair dist/*.whl
+    fi
+
+    deactivate
+}
+
+
+build_wheel_osx() {
+    echo "[BUILD WHEEL] Building with interpreter $1"
+    local skip=
+    setup_venv "$1"
+    (( $skip )) && return 0
```

Comment:
for clarity : how/when is this skipped?

Reply:
For bash nube like me:
```
local foo="" and local foo= are exactly equivalent. In both cases,
the right-hand side of the equal sign is an empty string.

local foo and local foo= are different: local foo leaves foo unset
while local foo= sets foo to an empty string. 
```

I see that `setup_venv` is doing check and setting the value.

Reply:
if version is less than 36, that comes out of `setup_venv`